packages/nimble-components/src/table/specs/table-action-menu-selection-hld.md

```diff
@@ -0,0 +1,234 @@
+# Table Selection Mode for Action Menus HLD
```

Comment:
The PR pipeline is failing in the lint step (see the "Checks" section at the bottom of the Conversation tab). This is an automated check that makes sure our documents have consistent formatting. To fix it you can follow [these steps in CONTRIBUTING.md](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#linting). If you haven't already, you'll first have to do the [getting started steps](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#getting-started) (install Node.js and run `npm install`).